sources/packages/backend/libs/services/src/health-check/health.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from "@nestjs/common";
+import { RedisOptions, Transport } from "@nestjs/microservices";
+import {
+  HealthIndicator,
+  HealthIndicatorResult,
+  HealthCheckError,
+  MicroserviceHealthIndicator,
+} from "@nestjs/terminus";
+import { ConfigService } from "@sims/utilities/config";
+import { ZeebeHealthIndicator } from "./zeebe-health-indicator";
+
+@Injectable()
+export class HealthService extends HealthIndicator {
+  constructor(
+    private microservice: MicroserviceHealthIndicator,
+    private readonly configService: ConfigService,
+    private readonly zeebeHealthIndicator: ZeebeHealthIndicator,
+  ) {
+    super();
+  }
+
+  /**
+   * Check the health of the application.
+   * @param key specifies if the health check is done for workers/queues.
+   * @returns status of the health check.
+   * In case of queues, we check if the redis is up and running.
+   * In case of workers, we check if the zeebe is ready to accept the connections.
+   */
+  async isHealthy(key: string): Promise<HealthIndicatorResult> {
+    let isHealthy;
```

Comment:
Those two variables are declared as any. Please check if your VS Code is properly alerting you about it.
![image](https://github.com/bcgov/SIMS/assets/61259237/63554040-a2aa-4369-8fa4-382af73e6951)

Reply:
yes it was complaining, missed it. I have updated now